tka/tailchonk.go

```diff
@@ -156,3 +165,255 @@ updateLoop:
 
 	return nil
 }
+
+// FS implements filesystem storage of TKA state.
+//
+// FS implements the Chonk interface.
+type FS struct {
+	base string
+	l    sync.RWMutex
```

Comment:
conventionally, please use "mu" instead of "l"

l also looks like I or 1 in some fonts

Reply:
Done!